backend/README.md

```diff
@@ -22,3 +22,5 @@ Run `stack exec -- edna-server -c dev-config.yaml` or `cabal run -- edna-server
 
 Set `EDNA_DEBUG_DB=1` to enable logging of all DB actions made.
 It works in tests as well.
+
+To collect some data with prometheus you can use `http://localhost:9000/metrics/` endpoint.
```

Comment:
```suggestion
To collect some data with prometheus you can use the `http://localhost:9000/metrics/` endpoint.
```

Reply:
Looks like there is no `the` in the end :smile: